sentry/src/main/java/io/sentry/ITransaction.java

```diff
@@ -69,10 +51,38 @@
    *
    * @return the event id
    */
-  @Nullable
+  @NotNull
   SentryId getEventId();
 
+  /**
+   * Attaches request information to the transaction.
+   *
+   * @param request the request
+   * @deprecated use {@link Scope#setRequest(Request)}
+   */
+  @Deprecated
+  @ApiStatus.ScheduledForRemoval
+  void setRequest(@Nullable Request request);
+
+  /**
+   * Returns the request information from the transaction
+   *
+   * @return the request or {@code null} if not set
+   * @deprecated use {@link Scope#getRequest()}
+   */
   @Nullable
-  @ApiStatus.Internal
-  String getTransaction();
+  @Deprecated
+  @ApiStatus.ScheduledForRemoval
+  Request getRequest();
```

Comment:
Do we need to add this here if it's already marked as deprecated?

Reply:
"ScheduledForRemoval" is a stronger variant of `Deprecated` and should be used together with `Deprecated`.